File: cpp/03_if.cpp
#include <iostream>
using namespace std;

int main() {
    int age = 20;

    if (age >= 18) {
        cout << "adult" << endl;
    } else {
        cout << "not adult yet" << endl;
    }

    int score = 85;

    if (score >= 90) {
        cout << "A" << endl;
    } else if (score >= 80) {
        cout << "B" << endl;
    } else if (score >= 70) {
        cout << "C" << endl;
    } else {
        cout << "need more work" << endl;
    }

    return 0;
}

File: rust/03_if.rs
fn main() {
    let age = 20;

    if age >= 18 {
        println!("adult");
    } else {
        println!("not adult yet");
    }

    let score = 85;

    if score >= 90 {
        println!("A");
    } else if score >= 80 {
        println!("B");
    } else if score >= 70 {
        println!("C");
    } else {
        println!("need more work");
    }
}
